examples/basic-example/README.md

```diff
@@ -0,0 +1,245 @@
+# Basic MarkdownDB tutorial
+
+In this tutorial, we'll learn how to use the MarkdownDB package to create a list of some of the cool projects we've built over the years and turn them into an SQLite database. Next, we'll also write a script, that will query a database for our projects and display them in a nice format on the terminal.
+
+## Step 1: Create a markdown file for each project
+
+First, let's create a markdown file for each project. You can use real project details or make up some examples. For the sake of this tutorial, we'll create files for some of the projects we've built at Datopian.
+
+```bash
+mkdir projects
+cd projects
+touch portaljs.md flowershow.md datapipes.md giftless.md data-cli.md aircan.md
+```
+
+In each file we'll write some short info about a given project, like so:
+
+```md
+# What is Portal.JS
```

Comment:
Very clever 👍 

BTW you could also have the default project be markdowndb (or one of the other ones). That way people learn about markdowndb as part of the tutorial